src/main/java/Calculator.java

```diff
@@ -0,0 +1,52 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Calculator {
+    public static void main(String[] args) {
+        String[] operationArray = {"+", "-", "/", "*"};
+        ArrayList<String>  operations = new ArrayList<>(Arrays.asList(operationArray));
```

Comment:
```suggestion
        List<String>  operations = new ArrayList<>(Arrays.asList(operationArray));
```

사용하지 않는 필드는 삭제해도 좋을 것 같군요.!
사용하지는 않지만 선언 및 초기화된 내용에 대해서 공유드릴 만한 내용이 있어서 짧은 설명과 관련된 링크 남깁니다.

자바는 인터페이스와 그를 구현한 구현체들이 존재합니다. 
[참고 - Collections in Java](https://www.javatpoint.com/collections-in-java)

컬렉션을 사용하는 방식은 공통적인 api인 입력, 조회, 수정, 삭제 등을 사용하고 상황에 따라 필요한 방식에 따라 적합한 자료구조를 선정하게 됩니다. 
그리하여 선언은 인터페이스(List)를 선언하고, 그에 따라 필요한 구현체(ArrayList)를 선언하는 방식으로 사용합니다. 

그리고 이펙티브 자바라는 책에서는 인터페이스를 타입으로 사용하는 습관을 길러 프로그램을 유연하게 사용하라는 내용이 있어 공유드립니다.

> "적합한 인터페이스만 있다면 매개변수뿐 아니라 반환값, 변수, 필드를 전부 인터페이스 타입으로 선언하라"

[이펙티브 자바 - 아이템 64 객체는 인터페이스를 사용해 참조하라](https://github.com/Meet-Coder-Study/book-effective-java/blob/main/9장/64_객체는_인터페이스를_사용해_참조하라_김재준.md)

Reply:
앗 감사합니다 
처음에 리스트 방식으로 할려다가 문자열 분리(split) 방식으로 바꾼 후 구문 삭제하는걸 깜박했어요,,,! 

+) 인터페이스 개념이 아직 낯설어서,,,아래 객체 분리 리뷰대로 적용해보고 도전해보겠습니다~!